Ensure dialog Password_needed is reached

hjluo opened the pull request:
##
### Ensure dialog Password_needed is reached

On Aarch64, we need to reset consoles to refresh the x11 console. sometimes the desktop runner will not launch up the password dialog.

- Related ticket: https://progress.opensuse.org/issues/169906
- Needles: N/A
- VR:
  - [**run 20 jobs with fix**](https://openqa.suse.de/tests/overview?distri=sle&build=hjlluo_withfix&version=15-SP7)
  - [**run 20 jobs without fix**](https://openqa.suse.de/tests/overview?distri=sle&build=hjlluo_no_change&version=15-SP7)

##

lemon-suse commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -60,9 +67,15 @@ sub run {
+    reset_consoles;
+    wait_still_screen 10;
 
+    handle_logout;
```
In VR code, you add 'script_run('free -g -h -t');', but in PR removed. Please confirmed before submit PR, and PR code should be same as VR.

hjluo replied:
it's just check the fre memory. it's like debug infor. so remove it.

lemon-suse commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -38,8 +39,9 @@ sub run {
     wait_screen_change { type_string "30" };
     wait_screen_change { send_key "alt-o" };
 
+    wait_still_screen 3;
```
Is this really needed?

hjluo replied:
removed.

hjluo commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -23,12 +23,13 @@ use warnings;
 
     # Password Settings
-    y2_module_guitest::launch_yast2_module_x11("security", match_timeout => 120, apply_workaround => is_sle('>=15-SP4') ? 1 : 0);
+    y2_module_guitest::launch_yast2_module_x11("security", match_timeout => 180, apply_workaround => is_sle('>=15-SP4') ? 1 : 0);
```
I'll set it back to 120

hjluo commented on schedule/yast/yast2_gui/yast2_gui_sle.yaml:
```diff
@@ -49,9 +49,9 @@ conditional_schedule:
         - yast2_gui/yast2_lan_restart_vlan
       aarch64:
         - console/setup_libyui_running_system
+        - yast2_gui/yast2_security
```
OK, change it back.

hjluo commented:
now [rerun 20 jobs](https://openqa.suse.de/tests/overview?version=15-SP7&distri=sle&build=hjluo&arch=aarch64) to verify if it still works after triming upper code.

jknphy commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -38,6 +39,11 @@ sub run {
+    reset_consoles;
+    wait_still_screen 10;
+
+    handle_logout;
+    handle_login;
```
why to logout and login back? Please in general explain all your changes in the description of the PR and commit.

hjluo replied:
Done.

jknphy commented:
In latest build the issue disappeared: https://openqa.suse.de/tests/16173454
We should try 20 runs also without any changes.

hjluo commented:
> In latest build the issue disappeared: [openqa.suse.de/tests/16173454](https://openqa.suse.de/tests/16173454) We should try 20 runs also without any changes.

now clone 16173454 with  [**no changes**](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP7&build=hjluo&arch=aarch64)

hjluo commented:
and with fix all `yast2_gui_with_fix_[1-20]` [passed](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP7&build=hjluo&arch=aarch64).

lemon-suse commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -37,7 +39,14 @@ sub run {
+        reset_consoles;
+        wait_still_screen 10;
 
+        handle_logout;
```
Could you confirm this logout/login is needed? Only needed between 1st and 2nd test, looks wired.

hjluo replied:
> Could you confirm this logout/login is needed? Only needed between 1st and 2nd test, looks wired.

OK, I'd run w/o this logout/login, currently looks we just need it at 1st and 2nd launch. Thanks.

hjluo replied:
@lemon-suse  Checked all ast2_gui_without_loginout[1-20] [passed](https://openqa.suse.de/tests/overview?build=hjluo&version=15-SP7&distri=sle&arch=aarch64). now remove the login/loginout.

hjluo replied:
@lemon-suse  Can you please take a look again the comment to see if it's proper to explain the fix.

jknphy commented:
could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?

hjluo commented:
> could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?

changed the comment in the PR to state this. thanks

hjluo commented on tests/yast2_gui/yast2_security.pm:
```diff
@@ -23,6 +23,8 @@ use warnings;
 use testapi;
 use version_utils qw(is_sle);
 use YaST::workarounds;
+use x11utils qw(handle_login handle_logout);
```
removed.

jknphy commented:
> > could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?
> 
> changed the comment in the PR to state this. thanks

Please, read again my comment, I'm not referring to code comments but to compare 20 runs with your change and 20 runs without it and add it to the first comment to the PR for visibility.

hjluo commented:
> > > could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?
> > 
> > 
> > changed the comment in the PR to state this. thanks
> 
> Please, read again my comment, I'm not referring to code comments but to compare 20 runs with your change and 20 runs without it and add it to the first comment to the PR for visibility.

You mean [this one](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20686#issuecomment-2538366568)?  I've changed it.

jknphy commented:
> > > > could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?
> > > 
> > > 
> > > changed the comment in the PR to state this. thanks
> > 
> > 
> > Please, read again my comment, I'm not referring to code comments but to compare 20 runs with your change and 20 runs without it and add it to the first comment to the PR for visibility.
> 
> You mean [this one](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20686#issuecomment-2538366568)? I've changed it.

your no-change test suites are from 3 days ago. Please show in your verification only 20 run with your change and 20 without, that would make more clear the situation, clean the rest of verification runs (login no login, etc.) those runs needs to be done with the same build and at the same point of time to make sense.

hjluo commented:
> > > > > could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?
> > > > 
> > > > 
> > > > changed the comment in the PR to state this. thanks
> > > 
> > > 
> > > Please, read again my comment, I'm not referring to code comments but to compare 20 runs with your change and 20 runs without it and add it to the first comment to the PR for visibility.
> > 
> > 
> > You mean [this one](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20686#issuecomment-2538366568)? I've changed it.
> 
> your no-change test suites are from 3 days ago. Please show in your verification only 20 run with your change and 20 without, that would make more clear the situation, clean the rest of verification runs (login no login, etc.) those runs needs to be done with the same build and at the same point of time to make sense.



> > > > > could you please add in your first comment a comparation of 20 runs without any fix to ensure that changes are still needed?
> > > > 
> > > > 
> > > > changed the comment in the PR to state this. thanks
> > > 
> > > 
> > > Please, read again my comment, I'm not referring to code comments but to compare 20 runs with your change and 20 runs without it and add it to the first comment to the PR for visibility.
> > 
> > 
> > You mean [this one](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20686#issuecomment-2538366568)? I've changed it.
> 
> your no-change test suites are from 3 days ago. Please show in your verification only 20 run with your change and 20 without, that would make more clear the situation, clean the rest of verification runs (login no login, etc.) those runs needs to be done with the same build and at the same point of time to make sense.

Rerun 20 jobs for withfix and without fix, and updated the VRs and the 1ste comments with same job [16173454](https://openqa.suse.de/tests/16173454)
